rustls/src/crypto/ring/suites.rs

```diff
@@ -0,0 +1,37 @@
+#[cfg(feature = "tls12")]
+use super::tls12::{
+    TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256, TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,
+    TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305_SHA256, TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,
+    TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384, TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305_SHA256,
+};
+use super::tls13::{
+    TLS13_AES_128_GCM_SHA256, TLS13_AES_256_GCM_SHA384, TLS13_CHACHA20_POLY1305_SHA256,
+};
+use crate::suites::SupportedCipherSuite;
+
+/// A list of all the cipher suites supported by rustls.
```

Comment:
Since `DEFAULT_CIPHER_SUITES` is based on `ALL_CIPHER_SUITES`, order `DEFAULT` before `ALL`?

Personally, I find this module on the small side and I would just keep these in `ring/mod.rs`. (They're also highly relevant so it makes sense to me for them to live there.)

Reply:
fixed the comment and hoisted this into `ring/mod.rs`